Paintroid/src/main/res/values/style.xml

```diff
@@ -152,4 +152,9 @@
         <item name="actionBarStyle">@style/ThemeOverlay.AppCompat.Dark.ActionBar</item>
     </style>
 
+    <style name="MoreOptionsMenuBackground" parent="ThemeOverlay.AppCompat.Light">
+        <item name="android:colorBackground">@android:color/white</item>
+        <item name="android:textColor">@android:color/black</item>
+    </style>
```

Comment:
you can move this into `PocketPaintTheme.Base` because the moreoptions menu should be white everywhere at any time. you can use `<item name="android:itemBackground">@android:color/white</item>` instead of `android:colorBackground` in the base theme. to keep it working.